internal/donor/donordata/enum_novoucherdecision.go

```diff
@@ -1,5 +1,4 @@
 // Code generated by "enumerator -type NoVoucherDecision -linecomment -empty"; DO NOT EDIT.
-
```

Comment:
you need to `go install ./cmd/enumerator`, I made a tiny change to add this empty line which prevents the docs taking the `Code generated by ...` line